### ISSUE TITLE
Can't get the ideal experiment results

### ISSUE THREAD

**Issue by weixmath** [jongchyisu/mvcnn_pytorch#1]
I use the parameters suggested in your paper and for vgg-11, I trained first stage for 30 epoch and second stage for 100 epoch. I got  91.76% per-class accuracy and 94.41% per-instance accuracy. I set learning rate 5e-5 and 1e-5 in first and second stages respectively and weight decay 0.001. Is there something wrong with my setting? Thank u.

**Comment by jongchyisu** [comment]
I have found the variance of the results is high. I did not report the variance in the paper, but I did get 94.4% to 95% ish test accuracy from different runs.

**Comment by weixmath** [comment]
OK,thank u. I just wanna know if it results from my mistakes. Good job :+1:
